libdnf5/base/transaction.cpp

```diff
@@ -292,6 +292,13 @@ void Transaction::Impl::add_resolve_log(
     logger.error(resolve_logs.back().to_string());
 }
 
+void Transaction::Impl::add_resolve_log(GoalProblem problem, SolverProblems problems) {
+    resolve_logs.emplace_back(LogEvent(problem, problems));
+    // TODO(jmracek) Use a logger properly
```

Comment:
Are you sure that my note is still applicable?

Reply:
I am not sure what you meant by that, so I didn't feel confident to remove it. But if you think it's resolved, I can remove all of them.